app/api_topologies.go

```diff
@@ -80,21 +80,24 @@ func updateFilters(rpt report.Report, topologies []APITopologyDesc) []APITopolog
 		ns = append(ns, namespace)
 	}
 	sort.Strings(ns)
-	for i, t := range topologies {
-		if t.id == podsID || t.id == servicesID || t.id == deploymentsID || t.id == replicaSetsID {
-			topologies[i] = updateTopologyFilters(t, []APITopologyOptionGroup{
-				kubernetesFilters(ns...), unmanagedFilter,
-			})
+	if len(ns) > 0 {
+		// We only want to apply k8s filters if any k8s info is present
```

Comment:
The purpose of the `len(ns) > 0` is to not show the k8s filters if there's no k8s-tracked nodes. I'll attempt to clarify.